app/controllers/api/users_controller.rb

```diff
@@ -24,7 +24,8 @@ def create
     user.update_facebook_avatar
     user.update_movies_from_facebook
 
-    render :json => user.to_json(:include => :elsewheres)
+    render :json => user.as_json(:include => :elsewheres)
+                        .merge(:has_completed_tour => user.recommendations.exists?)
```

Comment:
I think it'd be cleaner to create this as a method on `User` (`User#has_completed_tour`). And then we could include it in `as_json` by passing it in with the `methods` argument. 

From http://apidock.com/rails/ActiveModel/Serializers/JSON/as_json: 
> To include the result of some method calls on the model use :methods:

```
user.as_json(methods: :permalink)
# => { "id" => 1, "name" => "Konata Izumi", "age" => 16,
#      "created_at" => "2006/08/01", "awesome" => true,
#      "permalink" => "1-konata-izumi" }
```